mailer.js

```diff
@@ -61,8 +67,8 @@ module.exports = function (log) {
     this.mailer.close()
   }
 
-  Mailer.prototype._supportLinkAttributes = function () {
-    return linkAttributes(this.supportUrl)
+  Mailer.prototype._supportLinkAttributes = function (template) {
+    return linkAttributes(this._generateUTMLink(this.supportUrl, {}, template, 'support'))
```

Comment:
My original idea was that template would signify the email it came from and context would be what the user did in the email, [utm param breakdown](https://github.com/mozilla/fxa-auth-mailer/issues/182#issuecomment-234312264). For example, the support link in the account verified email would look something like this

`https://support.mozilla.org/kb/im-having-problems-with-my-firefox-account?utm_source=email&utm_medium=email&utm_campaign=fxa-welcome&utm_context=fxa-support`

That being said, there is probably a better naming convention for the values of the params.